openedx/core/lib/logsettings.py

```diff
@@ -126,6 +128,11 @@ def log_python_warnings():
     warnings.filterwarnings('ignore', 'Setting _field_data is deprecated')
     warnings.filterwarnings('ignore', 'Setting _field_data via the constructor is deprecated')
     warnings.filterwarnings('ignore', '.*unclosed.*', category=ResourceWarning)
+    warnings.filterwarnings('ignore', '.*You can remove default_app_config.*', category=RemovedInDjango41Warning)
```

Comment:
It looks like this exception was removed in Django 4.2. Let's use `PendingDeprecationWarning` instead and add an annotation that this can be removed after upgrading Django to 4.2.

Reply:
done